frontend/src/routes/Governance/components/GovernanceEmptyState.tsx

```diff
@@ -14,11 +14,11 @@ export function GovernanceCreatePolicyEmptyState() {
     return (
         <AcmEmptyState
             title={'Create policies to start monitoring cluster health'}
-            message={t('Use the button below to create policies.')}
+            message={t('Use the button below to create a policy.')}
```

Comment:
I suggest to make the following update: Use the following button to create a policy.